lib/ohai/plugins/windows/virtualization.rb

```diff
@@ -44,6 +44,14 @@
       virtualization[:system] = "kvm"
       virtualization[:role] = "guest"
       virtualization[:systems][:kvm] = "guest"
+    when "American Megatrends Inc."
```

Comment:
The next property that seems to be semi consistent is version.  On one version of hyper-v its value is 'VRTUAL - 5001223' and on another it is 'VRTUAL - 4001628'.  I have no idea what factors cause this to change. But, it seems messy to try to match on a partial string.